Orange/widgets/unsupervised/tests/test_owpca.py

```diff
@@ -205,9 +205,9 @@ def test_do_not_mask_features(self):
 
     def test_output_preprocessor(self):
         self.send_signal(self.widget.Inputs.data, self.iris)
-        pp = self.get_output(self.widget.Outputs.preprocessor)
-        self.assertIsInstance(pp, Preprocess)
-        transformed_data = pp(self.iris[::10])
+        template = self.get_output(self.widget.Outputs.transformed_data)
+        data = self.iris[::10]
+        transformed_data = data.transform(template.domain)
         self.assertIsInstance(transformed_data, Table)
         self.assertEqual(transformed_data.X.shape, (15, 2))
         output = self.get_output(self.widget.Outputs.transformed_data)
```

Comment:
Which function does this test test now? `data.transform`? :)

I suppose you can remove it. And also the one in test_owtsne; I believe (but may be wrong) that they just waste time.